tests/pages/components/task-list.js

```diff
@@ -0,0 +1,12 @@
+import {
+  collection,
+  text
+} from 'ember-cli-page-object';
+
+import TaskCard from 'code-corps-ember/tests/pages/components/task-card';
+
+export default {
+  title: text('[data-test-selector="task-list name"]'),
+
+  taskCards: collection(TaskCard)
```

Comment:
I figured you can create a collection with
```
collection({
  scope: 'selector',
  attributes...
});
```
so maybe putting the imported `TaskCard` page definition in there might work.